### PR TITLE
Making Icon Size Consistent

### LINKED ISSUE #2232: [UI] Icon size not consistent
**Describe the bug**
The icon size in the table is not the same.

**To Reproduce**

1. Go to https://firefox-ci-tc.services.mozilla.com/provisioners/mobile-3/worker-types/b-linux-large
2. Resize the window width until the icons under the First Claim and Worker ID column change

**Expected behavior**
The icons don't change in size when the window width changes.

**Actual behavior**

Screen Shot 2020-01-03 at 9 46 25 AM

**Taskcluster version**
24.1.1

### PR DESCRIPTION
**Pull request by aryamanpuri**
Resolves #2232

### CODE REVIEW

**Comment by aryamanpuri** [comment]
Screenshot from 2020-01-06 23-14-41

File: changelog/issue-2232 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+level: minor
```
Comment (helfi92) [inline changelog/issue-2232]: `silent`

File: ui/src/components/CopyToClipboardTableCell/index.jsx [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+import { withStyles } from '@material-ui/core/styles';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import TableCell from '@material-ui/core/TableCell';
 import ContentCopyIcon from 'mdi-react/ContentCopyIcon';
 import CheckIcon from 'mdi-react/CheckIcon';
 import TableCellItem from '../TableCellItem';
 import useCopyToClipboard from '../../utils/useCopyToClipboard';
 
+const styles = theme => ({
+  Icon: {
```
Comment (helfi92) [inline ui/src/components/CopyToClipboardTableCell/index.jsx]: No need to capitalize the class name.

**Comment by aryamanpuri** [comment]
Tests are failing?

**Comment by aryamanpuri** [comment]
Keeping the icon in the div element leads to the alignment problem

**Comment by helfi92** [comment]
> Keeping the icon in the div element leads to the alignment problem

We can try to fix that with CSS.

File: ui/src/components/CopyToClipboardTableCell/index.jsx [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
 import ContentCopyIcon from 'mdi-react/ContentCopyIcon';
 import CheckIcon from 'mdi-react/CheckIcon';
 import TableCellItem from '../TableCellItem';
 import useCopyToClipboard from '../../utils/useCopyToClipboard';
 
+const styles = theme => ({
+  icon: {
+    marginLeft: theme.spacing(1),
+    marginTop: theme.spacing(1),
+  },
```
Comment (helfi92) [inline ui/src/components/CopyToClipboardTableCell/index.jsx]: Using margin top to center things feels rather hacky and may not always work. Would the following work:

```diff
diff --git a/ui/src/components/CopyToClipboardTableCell/index.jsx b/ui/src/components/CopyToClipboardTableCell/index.jsx
index 8daa02c74..a676368ba 100644
--- a/ui/src/components/CopyToClipboardTableCell/index.jsx
+++ b/ui/src/components/CopyToClipboardTableCell/index.jsx
@@ -11,7 +11,7 @@ import useCopyToClipboard from '../../utils/useCopyToClipboard';
 const styles = theme => ({
   icon: {
     marginLeft: theme.spacing(1),
-    marginTop: theme.spacing(1),
+    display: 'flex',
   },
 });
```

Comment (aryamanpuri) [reply ui/src/components/CopyToClipboardTableCell/index.jsx]: Yes, you are right.
Resolved

**Pull request merged**